DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/compute_units/sample.json

```diff
@@ -8,6 +8,13 @@
   "targetDevice": ["FPGA"],
   "builder": ["ide", "cmake"],
   "languages": [{"cpp":{}}],
+  "commonFolder": {
+    "base": "../../..",
+    "include": [
```

Comment:
Do you think the list under "include" should also contain "Tutorials/DesignPatterns/compute_units"?

I think this is suggested by the following documentation (https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification):

`
"include": [
      "sample-folder-name",
`

> if the "commonFolder" object is present, it is expected to reference at least one folder, the sample folder

Reply:
Thanks Jessica, I agree.  I have added the compute units folder as you suggested.